tests/test_sequence.py

```diff
@@ -533,6 +533,64 @@ def test_ising_mode(
         seq2._in_ising = True
 
 
+@pytest.mark.parametrize("mappable_reg", [False, True])
+@pytest.mark.parametrize("parametrized", [False, True])
+def test_switch_register(reg, mappable_reg, parametrized):
+    pulse = Pulse.ConstantPulse(1000, 1, -1, 2)
+    seq = init_seq(
+        reg,
+        DigitalAnalogDevice,
+        "raman",
+        "raman_local",
+        [pulse],
+        initial_target="q0",
+        parametrized=parametrized,
+    )
```

Comment:
Do I get it right that you are not testing the conversion of a Sequence with a Mappable Register into a Sequence with a Register/another Mappable Register ? Is it worth testing ? Should we check what happens when DMM are configured in the Sequence as well ?

Reply:
You are correct. It doesn't hurt to check these cases, so I'll just add them